cogs/triggered/triggered.py

```diff
@@ -65,60 +66,68 @@ async def deepfry(self, ctx, user: discord.Member = None):
             user = ctx.message.author
         async with ctx.typing():
             # bot is typing here...
-            savePath = await self._createTrigger(user, mode=Modes.hypertriggered)
-            if not savePath:
+            data = await self._createTrigger(user, mode=Modes.hypertriggered)
+            if not data:
                 await self.bot.say("Something went wrong, try again.")
                 return
-            await ctx.send(file=discord.File(savePath))
+            await ctx.send(file=discord.File(data, filename=AVATAR_FILE_NAME.format(user)))
 
-    async def _createTrigger(self, user, mode=Modes.triggered):
+    async def _createTrigger(self, user: discord.User, mode=Modes.triggered):
```

Comment:
can we match the type hint with the docstring or vice versa?
```suggestion
    async def _createTrigger(self, user: discord.Member, mode=Modes.triggered):
```

Reply:
I will change the docstring to match the function signature.